lib/linguist/languages.yml

```diff
@@ -2675,6 +2695,15 @@ SQL:
   - .udf
   - .viw
 
+#IBM DB2
```

Comment:
Is this all commented out for a reason? If it's not required then please remove.